packages/concerto-core/changelog.txt

```diff
@@ -24,6 +24,8 @@
 # Note that the latest public API is documented using JSDocs and is available in api.txt.
 #
 
+Version 3.0.0 {2d4b45fa38a4db46d16c3520b0c56521} 2022-08-28
+
```

Comment:
Please increment the minor version number here and add a comment describing the change to api.txt file.

Reply:
Addressed ✅